framework/wazuh/core/agent.py

```diff
@@ -1361,6 +1348,7 @@ def get_manager_name() -> str:
     str
         Manager name.
     """
+    # TODO: This function needs to be redifined according to the required used case.
```

Comment:
We should get this value from the configuration (since it's not yet defined, we could add it to the technical debt issue)